src/mindi/mindi.c

```diff
@@ -136,16 +144,25 @@ void run_mindi( LV2_Handle handle, uint32_t nframes)
         //make event
         msg[0] = (((uint8_t)*plug->msgtype_p)<<4) + (uint8_t)(*plug->chan_p);
         //convert to note-off if appropriate
-        if(!*plug->enable_p && plug->notesent && *plug->msgtype_p == 0x09)
+        if(((!*plug->moment_p && plug->momentstate) || !*plug->enable_p) && plug->notesent && *plug->msgtype_p == 0x09)
```

Comment:
This gets AUTOFF working in a reasonable manner for momentary notes (while making the fewest changes to existing code).  My addition here is saying when the momentary button transitions from ON to OFF, a previously sent note will produce a note-off.